CHANGELOG.md

```diff
@@ -1,3 +1,103 @@
+### citus v11.2.0 (January 30, 2023) ###
+
```

Comment:
What about MERGE support? `prepare_chanelog.pl` script couldn't find any "description:" lines for it. And moreover, we're partially reverting that feature in #6675?

Reply:
we still have MERGE for Citus local tables